server/models/actionsModel.js

```diff
@@ -0,0 +1,16 @@
+const actionId = 'actionId'
```

Comment:
I find this whole "model" very odd, but that may just be my lack of familiarity with Hapi and Nunjucks. Happy to leave it as-is for now as it matches the web service.

Reply:
I found it weird as well, but as it was already in use in the web front end, just went with it in the interests of time. It was working ok there, but didn't match the requirements of the admin front end so I didn't end up using it.
Definitely something we need to have a discussion about when we've got more time.